common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -44,6 +49,49 @@ public class CommonValue {
             + "      \""
             + MLModel.MODEL_CONTENT_FIELD
             + "\" : {\"type\": \"binary\"},\n"
+            + "      \""
+            + MLModel.CHUNK_NUMBER_FIELD
+            + "\" : {\"type\": \"long\"},\n"
+            + "      \""
+            + MLModel.TOTAL_CHUNKS_FIELD
+            + "\" : {\"type\": \"long\"},\n"
+            + "      \""
+            + MLModel.MODEL_ID_FIELD
+            + "\" : {\"type\": \"keyword\"},\n"
+            + "      \""
+            + MLModel.DESCRIPTION_FIELD
+            + "\" : {\"type\": \"text\"},\n"
+            + "      \""
+            + MLModel.MODEL_FORMAT_FIELD
+            + "\" : {\"type\": \"keyword\"},\n"
+            + "      \""
+            + MLModel.MODEL_STATE_FIELD
+            + "\" : {\"type\": \"keyword\"},\n"
+            + "      \""
+            + MLModel.MODEL_CONTENT_SIZE_IN_BYTES_FIELD
+            + "\" : {\"type\": \"long\"},\n"
+            + "      \""
+            + MLModel.MODEL_CONFIG_FIELD
+            + "\" : {\"properties\":{\""
+            + MODEL_TYPE_FIELD + "\":{\"type\":\"keyword\"},\""
+            + EMBEDDING_DIMENSION_FIELD + "\":{\"type\":\"long\"},\""
```

Comment:
[nit]Does this field need to be `long`?

Reply:
Good catch, that part should be int. Will keep this backport PR as is, and change in next PR.